src/file_constants.h

```diff
@@ -4,6 +4,7 @@
 #include <sys/stat.h>
 #include <fcntl.h>
 const JL_DUMMY = 0
+const JL_O_ACCMODE     = O_ACCMODE
```

Comment:
This needs `ifdef O_ACCMODE`, but I am not certain it belongs in the list (it is the bitwise OR of WRONLY RDONLY and RDRW)